jfrog-cli/artifactory/utils/buildinfoproperties.go

```diff
@@ -193,6 +207,25 @@ func CreateBuildInfoPropertiesFile(buildName, buildNumber string, config *viper.
 	return propertiesFile.Name(), nil
 }
 
+// If HTTP_PROXY env variable presented, adds to the config to deploy Gradle/Maven artifacts via proxy
+func setProxyIfDefined(config *viper.Viper) error {
```

Comment:
In addition to the HTTP_PROXY env var, we should probably also support the *HTTPS_PROXY* env var.
See the *TestArtifactoryProxy* as a reference.

Reply:
We don't support https in the build info. We only support http. I've checked and the proxy is usually running on http only without https.